MaxText/standalone_checkpointer.py

```diff
@@ -70,20 +72,18 @@ def checkpoint_loop(config, state=None):
 
   jax.block_until_ready(state)
   checkpoint_load_end = datetime.datetime.now()
-  checkpoint_read_time = (checkpoint_load_end - checkpoint_load_start).total_seconds()
-  ckpt_read_time.append([jax.process_index(), checkpoint_read_time])
   
   if state is not None:  # Checkpoint was available for restore
     if jax.process_index() == 0:
-      max_logging.log(f"STANDALONE CHECKPOINTER : Checkpoint restored in : {checkpoint_load_end - checkpoint_load_start}")
+      max_logging.log(f"STANDALONE CHECKPOINTER : Initial checkpoint restored in : {checkpoint_load_end - checkpoint_load_start}")
   else:  # Checkpoint was unavailable, state needs to be initialized
     state, _, _ = max_utils.setup_training_state(model, None, tx, config, init_rng, mesh, checkpoint_manager)
   state = add_entropy_to_checkpoint(state)
 
   start_step = get_first_step(state)  # this is the start_step for training
   for step in np.arange(start_step, config.steps):
+    start_time = datetime.datetime.now()
```

Comment:
can we move this tighter after the "barrier before save"?

Reply:
Done, added a point to update it there. I left the declaration here too since the code will still use it if checkpoint_manager is None, though I'm not sure when that could ever happen